README.md

```diff
@@ -70,7 +70,8 @@
 
```

Comment:
- [ ] Your acknowledgement section of the readme file should have a credit to Gregoire Vella whose design was used.